xml_inscriptions/transcribed/ITA.Rome.AAR.L.9262.xml

```diff
@@ -175,8 +176,8 @@
 
     <facsimile>
         <surface>
-            <desc>Image from the AAR Archaeology Archive</desc>
-            <graphic url="http://nvb.aarome.org/privato/articoli/1066_2.JPG"/>
+            <desc/>
```

Comment:
Do we want to get rid of the AAR's image and just leave our own? That's fine, I just want to be sure before we remove it.

Reply:
That was my idea, I thought since we have our own image that would be preferable; also, I don't think it was displaying properly, but I am open to different suggestions!